src/components/configEditor.tsx

```diff
@@ -34,6 +34,8 @@ const enableTemplatesTooltip = `Enable the templates tab for use with the Cognit
 
 const enableEventsAdvancedFilteringTooltip = `Enable the Events advanced filtering`;
 
+const enableFlexibleDataModellingTooltip = 'Enable Flexible Data Modelling';
```

Comment:
```suggestion
const enableFlexibleDataModellingTooltip = 'Enable Flexible Data Modelling (preview)';
```
and please add the same `(preview)` to `enableEventsAdvancedFilteringTooltip` as well